### ISSUE TITLE
OTLP Exporter Configuration - missing some metrics environment variables

### ISSUE THREAD

**Issue by aabmass** [open-telemetry/opentelemetry.io#5095]
https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/ documents OTLP environment variables but is missing the variables from the Metrics SDK spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md

- `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE`
- `OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION`

**Comment by theletterf** [comment]
+CC @open-telemetry/collector-approvers

**Comment by mx-psi** [comment]
@theletterf I think you meant to ping a different handle? I don't think this has to do with the Collector

**Comment by theletterf** [comment]
@mx-psi You're right! Would spec-semconv be the most appropriate here, @svrnm ?

**Comment by svrnm** [comment]
I would say @open-telemetry/spec-sponsors ?

**Comment by cartermp** [comment]
The general thinking behind this page (when I first wrote it, that is) is that it documents some of the most-used env vars. But it hasn't really changed since then, and this was before metrics stabilized. So I'm in favor of just adding these.

**Comment by svrnm** [comment]
> Sorry, what is the ask from spec-sponsors? This just looks like a doc gap, and can be fixed by doc owners.

From my point of view, Spec sponsors like other approvers&maintainers are doc owners

**Comment by cijothomas** [comment]
> > Sorry, what is the ask from spec-sponsors? This just looks like a doc gap, and can be fixed by doc owners.
> 
> From my point of view, Spec sponsors like other approvers&maintainers are doc owners

I see. Sorry I wasn't seeing it that way. I assumed that docs are owned by docs team and content is sourced from the spec repo. I think Spec sponsors' responsibility is to make any changes/clarifications to the spec itself. Since this issue is simply missing some configuration in the docs page, I was wondering, what help is needed from spec-sponsors. Doc owners can just make the change to the doc.

(I am not opposed to making spec-sponsors as doc owners. Its best to call it out explicitly if that is the expectation)

**Comment by svrnm** [comment]
> > > Sorry, what is the ask from spec-sponsors? This just looks like a doc gap, and can be fixed by doc owners.
> > 
> > 
> > From my point of view, Spec sponsors like other approvers&maintainers are doc owners
> 
> I see. Sorry I wasn't seeing it that way. I assumed that docs are owned by docs team and content is sourced from the spec repo. I think Spec sponsors' responsibility is to make any changes/clarifications to the spec itself. Since this issue is simply missing some configuration in the docs page, I was wondering, what help is needed from spec-sponsors. Doc owners can just make the change to the doc.
> 
> (I am not opposed to making spec-sponsors as doc owners. Its best to call it out explicitly if that is the expectation)

Yeah, my assumption here was implicit, we might want to clarify that, but that's a separate issue, not related to this discussion.

**Comment by cijothomas** [comment]
Yes that is separately clarifiable.
For this issue, I support what Carter suggested: https://github.com/open-telemetry/opentelemetry.io/issues/5095#issuecomment-2331539329 but it'd be good to call out that ENV vars are optional and languages may/may not have it to avoid surprises. (I believe this was also reported by some users in the past, but I cannot find the tracking issue now.)

**Comment by cijothomas** [comment]
related : https://github.com/open-telemetry/opentelemetry-specification/issues/3929

**Comment by cijothomas** [comment]
https://github.com/open-telemetry/opentelemetry.io/pull/4099/files can be extended to cover this page as well https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/ to close my concern!
